src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -43,7 +43,7 @@
     <Delete Files="%(_ReferenceSerializerIntermediateFolder.Identity).cs" ContinueOnError="true"/>
     <Delete Files="%(_ReferenceSerializerIntermediateFolder.Identity).pdb" ContinueOnError="true"/>
     <Message Text="Running Serialization Tool for Reference Assembly" Importance="normal" />
-    <Exec Command="dotnet Microsoft.XmlSerializer.Generator --force --quiet --reference @(Reference) --assembly %(_TargetSerializationAssembly.Identity) --type %(_TargetSerializationAssembly.SerializationTypes) --out $(IntermediateOutputPath)" ContinueOnError="true" />
+    <Exec Command="dotnet Microsoft.XmlSerializer.Generator --force --quiet --reference &quot;@(Reference)&quot; --assembly %(_TargetSerializationAssembly.Identity) --type %(_TargetSerializationAssembly.SerializationTypes) --out $(IntermediateOutputPath)" ContinueOnError="true" />
```

Comment:
I can add quotes around %(_TargetSerializationAssembly.Identity) but not $(IntermediateOutputPath) since the latter is a relative path. It won't work if add quote on a relative path.